dotnet/src/webdriver/Interactions/ActionSequence.cs

```diff
@@ -73,7 +73,7 @@ public int Count
         /// </summary>
         public InputDevice inputDevice
```

Comment:
Hm, while we are here, let's mark this property obsolete, and introduce new one **I**nputDevice.

Reply:
That's a good point. We can't be breaking anyone, since getting this value will always blow up.

Reply:
![image](https://github.com/user-attachments/assets/1b1cc428-df40-418b-88e6-a9bb8c7e532d)
This is apparently not CLS-compliant. Should it just be a breaking change?

Reply:
Up to you, you can make it as separate PR, but this PR will be dependent on new on. I guess we are close to merge it.

Reply:
So just rename :D

Reply:
I made the change in this PR, it should be ready now.

Reply:
Was thinking... It is still a breaking change at source level. If user doesn't invoke it at runtime, it doesn't mean we will not break him.

It is better to be safer: obsolete old and introduce new. Thanks for understanding.

Reply:
✅ now we have two 
![image](https://github.com/user-attachments/assets/bef09a01-bc82-4b24-ad1c-f9b2f5d60d4d)

Reply:
Yeap, you already chose this way. 👍